src/fides/api/schemas/connection_configuration/connection_config.py

```diff
@@ -66,6 +66,9 @@ def mask_sensitive_fields(
     if connection_secrets is None:
         return connection_secrets
 
+    if not secret_schema.get("properties"):
```

Comment:
Maybe add a comment here explaining why it returns if there are no properties in the secret schema

Reply:
Comment added!